healthtools_ke_api/views/search.py

```diff
@@ -33,7 +33,7 @@ def remove_keyword(query):
         Remove keyword from search term
         """
         query_formatted = query.strip().lower()
-        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o', 'clinical officer','health facility']
+        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o','hf', 'h.f', 'h.f.', 'clinical officer','health facility']
```

Comment:
Let's cluster these appropriately i.e `'clinical officers'` should come before the health facility alternatives. We don't simply do.. we do well.